IdentityCore/src/webview/embeddedWebview/ui/MSIDWebviewUIController.h

```diff
@@ -46,6 +46,7 @@ NSWindowController
 #if TARGET_OS_IPHONE
 @property (nonatomic, weak) UIViewController *parentController;
 @property (nonatomic) UIModalPresentationStyle presentationType;
+@property (nonatomic) BOOL presentInParentController;
```

Comment:
You also need to update macOS implementation for the same

Reply:
Also, I'm a bit confused. Why don't you just pass in webview?

Reply:
I just double checked, the macOS version of MSIDWebviewUIController doesn't have presentInParentController.

For your second question - I need to overwrite the -(void)presentView method in MSAIWebviewUIController and MSAIOAuth2EmbeddedWebviewController in MSAL CPP, so I need access to presentInParentController in MSAL CPP.

original:
<img width="907" alt="b2811861c1c412cd649245f20055911" src="https://user-images.githubusercontent.com/62267180/182200807-13cc2bdf-46d7-4e6d-867e-6395f71473c7.png">

new:
<img width="885" alt="cf118b6fc3b29ee82dfb4b7a2ecc5cc" src="https://user-images.githubusercontent.com/62267180/182201284-7e1d9f09-caf1-424c-ba33-9dfac3d64104.png">

Office is doing this manually every time after they consume OneAuth/MSAL, and we have an email thread that OneAuth/MSAL should do this for all partners.

Reply:
Ok, then you need to make this property readonly in the public header file + mark as iOS only. You can keep it as a readwrite in the .m file.

Reply:
@oldalton, thanks!